src/modules/70-pipeline/pages/webhooks/WebhookEvents/WebhooksEventsList.tsx

```diff
@@ -33,7 +38,7 @@ export type CustomColumn<T extends Record<string, any>> = Column<T>
 export default function WebhooksEventsList({
   response
 }: {
-  response: ListGitxWebhookEventsOkResponse | undefined
+  response: ResponseWithPagination<ListGitXWebhookEventResponseResponse> | undefined
```

Comment:
Not a big deal, but better to set as optional rather than potentially `undefined`
```suggestion
  response?: ResponseWithPagination<ListGitXWebhookEventResponseResponse>
```